packages/video_player/video_player/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android, iOS, and web.
 repository: https://github.com/flutter/plugins/tree/main/packages/video_player/video_player
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
-version: 2.2.18
+version: 2.3.0
 
```

Comment:
You need to update the video_player_platform_interface minimum version to require the version that added the new option. As written someone's project could pass through package resolution with a version that wouldn't compile.